README.md

```diff
@@ -1,2 +1,205 @@
-# ignition_dev_tinkering
-Testing out ways to create user-friendly development environments for https://github.com/ignitionrobotics
+# Ignition Docker
+
+A user-friendly environment for work related to [Ignition Robotics](https://github.com/ignitionrobotics).
+
+## About
+
+Often times, when developing, testing, or using new features in Ignition libraries, setting up the environment can be cumbersome.
+This repository is meant to provide Ignition users with a framework that requires minimal configuration for development, testing and usage of Ignition libraries.
+
+The approach taken here is motivated by [Docker](https://www.docker.com/).
+The idea is to provide users with a Docker container that contains a binary install of the Ignition libraries, along with tools like [colcon](https://colcon.readthedocs.io/en/released/) that can be used for building a user's source code.
+If users need features that aren't available in the Ignition binaries, they can load their own copy of the Ignition repositories into the Docker container, and then build these repositories inside of the container before using Ignition.
```

Comment:
What about providing a version of the dockerfile that builds on this one to provide a source Ignition workspace ready to go?

Reply:
This was discussed a bit previously, but we decided to move away from this due to the reasoning behind this [comment](https://github.com/ignitionrobotics/ign-gazebo/pull/397#issuecomment-713680445) and the discussion in this [issue](https://github.com/ignitionrobotics/ign-common/issues/112).